incubator/haproxy-ingress/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
 name: haproxy-ingress
-version: 0.0.27
+version: 0.0.28
 appVersion: 0.7.2
 home: https://github.com/jcmoraisjr/haproxy-ingress
-description: Ingress controller implementation for haproxy loadbalancer.
+description: DEPRECATED! See https://github.com/haproxy-ingress/charts
```

Comment:
See https://github.com/helm/charts/blob/master/PROCESSES.md for required changes when deprecating a chart.

Reply:
Missed that, thanks for the review. Updated.